docs/source/reading_writing.rst

```diff
@@ -5,16 +5,17 @@ Reading and writing
 
 .. _reading chains:
 
-Reading chain files from PolyChord, MultiNest, CosmoMC, or Cobaya
-=================================================================
+Reading chain files from PolyChord, MultiNest, UltraNest, CosmoMC, or Cobaya
+============================================================================
 
-`MultiNest <https://github.com/farhanferoz/MultiNest>`_
+If you have finished nested sampling or MCMC runs from one of:
+
+* `PolyChord <https://polychord.io>`_: https://github.com/PolyChord/PolyChordLite
+* MultiNest: https://github.com/farhanferoz/MultiNest
+* `UltraNest <https://johannesbuchner.github.io/UltraNest/index.html>`_: https://github.com/JohannesBuchner/UltraNest
+* `CosmoMC <https://cosmologist.info/cosmomc/readme.html>`_: https://github.com/cmbant/CosmoMC
+* `Cobaya <https://cobaya.readthedocs.io>`_: https://github.com/CobayaSampler/cobaya
```

Comment:
Have we checked that the change from `-` `*` doesn't change/improves the formatting?

Reply:
This wasn't a change from `-` to `*` (don't confuse the GitHub minus with a `-`...). I changed prose to an itemised list.